namespaces/live.cloud-platform.service.justice.gov.uk/stability-inspector-dev/resources/variables.tf

```diff
@@ -0,0 +1,87 @@
+variable "vpc_name" {
+}
+
+variable "kubernetes_cluster" {
+}
+
+variable "eks_cluster_name" {
+  description = "The name of the eks cluster to retrieve the OIDC information"
+}
+
+variable "application" {
+  description = "Name of Application you are deploying"
+  type        = string
+  default     = "Justice Gov UK Archive"
+}
+
+variable "namespace" {
+  description = "Name of the namespace these resources are part of"
+  type        = string
+  default     = "stability-inspector-dev"
+}
+
+variable "archive_date" {
```

Comment:
Probably left over from the archiver repo?

Reply:
Thank you :)